src/ralph/sim_cards/models.py

```diff
@@ -0,0 +1,41 @@
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+from django.conf import settings
+
+from ralph.assets.models.base import BaseObject
+from ralph.lib.mixins.models import AdminAbsoluteUrlMixin
+from ralph.back_office.models import Warehouse
+from ralph.lib.mixins.models import NamedMixin
+
+
+class CellularCarrier(AdminAbsoluteUrlMixin, NamedMixin, models.Model):
+    pass
+
+class SIMCard(AdminAbsoluteUrlMixin, models.Model):
+    pin1 = models.CharField(max_length=8)
+    puc1 = models.CharField(max_length=10)
+    pin2 = models.CharField(max_length=8, null=True, blank=True)
+    puc2 = models.CharField(max_length=10, null=True, blank=True)
+    carrier =models.ForeignKey(
+        CellularCarrier, on_delete=models.PROTECT, null=True,
+    )
+    card_number = models.CharField(max_length=20, unique=True)
+    phone_number = models.CharField(max_length=15, unique=True)
+    warehouse = models.ForeignKey(Warehouse, on_delete=models.PROTECT)
+    owner = models.ForeignKey(
+        settings.AUTH_USER_MODEL, null=True, blank=True,
+        related_name='owned_simcards',
+    )
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL, null=True, blank=True,
+        related_name='used_simcards',
```

Comment:
used or user?

Reply:
I use the same name:
Used for used_simcards 
Owned for owned_simcards